Meadowlark-js/packages/meadowlark-core/src/validation/PaginationValidator.ts

```diff
@@ -8,7 +8,7 @@ import { PaginationParameters } from '../message/PaginationParameters';
 import { createInvalidRequestResponse } from '../Utility';
 
 const isNotPositiveInteger = (value: string | number): Boolean =>
-  !Number(value) || (R.is(Number, value) ? value : Number.parseInt(value, 10)) < 1;
+  !Number(value) || (R.is(Number, value) ? value : Number.parseInt(value.toString(), 10)) < 1;
```

Comment:
@bradbanister when running the build inside the Docker container - which is Debian based - this line was failing, with TypeScript complaining that `Number.parseInt` can only take a string, not a number. Same version of TypeScript, slightly newer version of Node than what I have on my computer (16.17 vs 16.14, if I remember correctly).